tslint.json

```diff
@@ -24,6 +24,7 @@
     "array-type": [true, "array"],
     "one-line": false,
     "no-any-union": false,
-    "void-return": false
+    "void-return": false,
+    "unified-signatures": false
```

Comment:
With the introduction of `unified-signatures: false`, I believe these can be safely removed now:

https://github.com/discordjs/discord.js/blob/9d7152f0fdcd879cad654fe4fdd658fac57ceb02/typings/index.d.ts#L2341-L2345

https://github.com/discordjs/discord.js/blob/9d7152f0fdcd879cad654fe4fdd658fac57ceb02/typings/index.d.ts#L3095-L3099